### PR TITLE
docs/19550-candlestick-documentation-improvement

### PR DESCRIPTION
**Pull request by jedrzejruta**
Enhanced candlestick documentation with information about candles wick and body colors altering.

### CODE REVIEW

File: docs/stock/candlestick-chart.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+}
+```
+
+Properties given for candlestick colors are:
+- `color` Candle body color when going down (closing value is lower than the opening value)
+- `lineColor` color of the wick when going down (closing value is lower than the opening value) m
+- `upColor` Candle body color when going up (closing value is higher than the opening value), by default it is transparent
+- `upLineColor` Color of the wick when going up (closing value is higher than the opening value)
+
+![candlestick-color.png](candlestick-color.png)
```
Comment (pawelfus) [inline docs/stock/candlestick-chart.md:30]: Can we replace the image with an embedded live demo? I think [this one](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/plotoptions/candlestick-linecolor/) would be good. Feel free to update colors, to match the config above

**Review by pawelfus — CHANGES_REQUESTED** [review]
Minor improvements, overall great, thanks!

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**